src/WebJobs.Script.WebHost/App_Start/WebApiConfig.cs

```diff
@@ -117,7 +117,7 @@ private static WebHostSettings GetDefaultSettings()
             bool isLocal = string.IsNullOrEmpty(home);
             if (isLocal)
             {
-                settings.ScriptPath = Environment.GetEnvironmentVariable("AzureWebJobsScriptRoot");
```

Comment:
Looks like you're reverting a change I made - probably because you're not rebased on dev?